app/views/returns/ExportedHumanMedicinesPlasticPackagingView.scala.html

```diff
@@ -1,68 +0,0 @@
-@*
- * Copyright 2022 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@import uk.gov.hmrc.govukfrontend.views.html.components._
-@import viewmodels.PrintLong
-@import components._
-
-@this(
-    layout: templates.Layout,
-    formHelper: FormWithCSRF,
-    govukErrorSummary: GovukErrorSummary,
-    govukRadios: GovukRadios,
-    govukButton: GovukButton,
-    paragraphBody: paragraphBody,
-    sectionHeader: SectionHeader,
-    details: GovukDetails
-)
-
-@(amount: Long, form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
-
-@paragraphContent ={
-    @paragraphBody(messages("humanMedicinesPlasticPackaging.detail"))
-
-    @details(Details(
-        id = Some("reveal"),
-        summary = Text(messages("humanMedicinesPlasticPackaging.reveal")),
```

Comment:
these need to go from the messages file :)

Reply:
Carl done that